### PR TITLE
Add Secret Santa draw

### PR DESCRIPTION
**Pull request by mariocj89**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -864,4 +897,29 @@ components:
...
                   - TAIL
-
+    SecretSanta:
+      type: object
+      properties:
+        langauge:
+          type: string
+          enum:
+            - es
+            - en
```
Comment (dnaranjo89) [inline swagger.yaml:907]: could we use `es-ES` and `en-GB`? these are the locale types we are using in the frontend

File: swagger.yaml [hunk 2]
```diff
@@ -864,4 +897,29 @@ components:
...
+        participants:
+          type: array
+          items:
+            maxLength: 500
+            minLength: 3
+            type: object
+              properties:
```
Comment (dnaranjo89) [inline swagger.yaml:914]: We don't have exclusions here but I see they are mentioned somewhere else in the PR

File: swagger.yaml (hunk 1 above)
Comment (mariocj89) [reply swagger.yaml:907]: ehhhm, isnt `es-ES` specific to spain whilst `es` denotates all Spanish speaking languages? :/ same for en.

File: swagger.yaml (hunk 2 above)
Comment (mariocj89) [reply swagger.yaml:914]: Yeah, I did not added in the first iteration to unblock you ASAP.

Comment (mariocj89) [reply swagger.yaml:914]: Exclusions are ready now.

File: swagger.yaml (hunk 1 above)
Comment (dnaranjo89) [reply swagger.yaml:907]: ah yeah, I though the standard was always 4 letter but it's not. 

btw check this one 😂 
https://github.com/unicode-org/cldr-json/blob/f321d7704452b076a0f75a988889cfdb87802f35/cldr-json/cldr-localenames-full/main/en-GB/languages.json#L178

**Pull request merged**